client/src/components/profile/editprofile.js

```diff
@@ -187,6 +274,240 @@ class EditProfilePopup extends Component {
               {alertFailure.msg}
             </Alert>
           }
+          <img
+            className="edit-profile-popup__user-img"
+            src={`https://avataaars.io/?avatarStyle=Circle&topType=${top}&accessoriesType=${accessories}&hairColor=${hairColor}&facialHairType=${facialHair}&clotheType=${clothes}&eyeType=${eyes}&eyebrowType=${eyebrow}&mouthType=${mouth}&skinColor=${skin}&clotheColor=${clothColor}`}
+            alt="your current profile"
+            />
+
+          {/* Avatar Properties */}
```

Comment:
Create a separate file for edit-avatar inside `profile` folder, and refactor all the avatar-related boilerplate. You can also make the state separate as well, by making a completely different button for the edit avatar `pencil icon` attached to the profile pic.

So, Edit Profile will only edit the full name, country, etc, whereas Pencil Icon (edit avatar button attached next to profile pic) will change only avatar.

Both will hit the same endpoint for now - PUT `/edit-profile`

NOTE: Separating state is optional, whichever works best just proceed with it but you do need to implement the separate button and refactor the boiler plate.